src/main/kotlin/Main.kt

```diff
@@ -1,11 +1,23 @@
-import Calculator.Calculator
+import lotto.InputView
+import lotto.LottoGame
+import lotto.ResultView
```

Comment:
관련된 클래스들을 패키지로 모아서 관리할 수 있도록 해보세요. 아래의 글을 참고하세요.
https://edu.nextstep.camp/s/Wb0CSsVb/ls/GFWaks9v

```suggestion
import lotto.domain.LottoGame
import lotto.view.InputView
import lotto.view.ResultView
```